roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -12,5 +12,12 @@
 - name: start aci-gw container
   service: name=aci-gw state=started
 
+- name: Contiv | Wait for netmaster
```

Comment:
may be the task name should be `wait for netmaster to be ready` (without `Contiv |` prefix) to be consistent with rest of tasks?